canonical/canonical.com
# Build diversity page

## Done

- Build careers diversity page
- Add test
- Redirect old diversity page to new diversity page
- Added item to navigation
- Remove old diversity pages

## QA

- Compare https://canonical-com-840.demos.haus/careers/company-culture/diversity with [design](https://www.figma.com/file/qCxGiBTgSFa21iVR7YsgWO/%2FCareers%2FDiversity?node-id=18-2374&t=2Naxc688KVcybL9u-0)
- Compare https://canonical-com-840.demos.haus/careers/company-culture/diversity with [copy docs](https://docs.google.com/document/d/1l1UGJToc0ErHUMN9W8hELvF5sDyVRO7N0xDzncqA2SQ/edit#heading=h.ov7lrxgwh2dv)
- https://canonical-com-840.demos.haus/careers/diversity should redirect to https://canonical-com-840.demos.haus/careers/company-culture/diversity 
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-179?atlOrigin=eyJpIjoiMTkwYjc4Y2M5NDkyNDc3MzlhMGZmZDI0MzM1NTZjODAiLCJwIjoiaiJ9
